<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindOrRepalce</class>
 <widget class="QWidget" name="FindOrRepalce">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查找/替换</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>查找:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="findLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>替换为:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="replaceLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="previous">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>上一个</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="next">
       <property name="text">
        <string>下一个</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="replace">
       <property name="text">
        <string>替换</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="replaceAll">
       <property name="text">
        <string>替换全部</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
